homeassistant/components/geonetnz_quakes/strings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "config": {
+    "title": "GeoNet NZ Quakes",
+    "step": {
+      "user": {
+        "title": "Fill in your location and filter details.",
+        "data": {
+          "latitude": "Latitude",
```

Comment:
Please update the strings.

Reply:
OK, removed all unused strings.

Reply:
Please run the translation develop script again.

Reply:
OK, included now.